config/puma.rb

```diff
@@ -0,0 +1,21 @@
+# https://devcenter.heroku.com/articles/deploying-rails-applications-with-the-puma-web-server
+
+workers Integer(ENV['WEB_CONCURRENCY'] || 1)
```

Comment:
That is strange. I have an instance of Ohana API running on 2 Heroku dynos with 2 Puma workers and 5 Puma threads, and memory usage rarely goes above 340MB.